tests/GraphQl/Resolver/Stage/ReadStageTest.php

```diff
@@ -221,6 +222,45 @@ public function testApplyCollection(array $args, ?string $rootClass, ?array $sou
         $this->assertSame($expectedResult, $result);
     }
 
+    /**
+     * Test a regression where the order in which order fitlers were applied
+     * was wrong when the data had been ordered by a nested field.
+     */
+    public function testPreserveOrderOfOrderFitlersIfNested()
```

Comment:
```suggestion
    public function testPreserveOrderOfOrderFiltersIfNested(): void
```

Reply:
A classic `Fitler`. Missed that one, sorry. Applied 👍🏽